specification/web/resource-manager/Microsoft.Web/2016-03-01/ResourceProvider.json

```diff
@@ -307,6 +307,51 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/verifyHostingEnvironmentVnet": {
+      "post": {
+        "summary": "Verifies if this VNET is compatible with an App Service Environment.",
+        "description": "Verifies if this VNET is compatible with an App Service Environment.",
+        "operationId": "VerifyHostingEnvironmentVnet",
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml",
+          "application/x-www-form-urlencoded"
+        ],
+        "produces": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml"
+        ],
```

Comment:
@hforeste still the same question, does your service support all of the content types (text/json, application/xml)? If supported content type are the same, could you place them to the top level of the swagger?
@ravbhatnagar are you ok with these content types?

Reply:
@hforeste - ARM does not support content types other than application/json. So, please remove them. Even though these are generated by a tool, will it be a problem if you manually fix these?

Reply:
No, it should not be a problem. Thanks!